shell.nix

```diff
@@ -0,0 +1,22 @@
+{ 
+  pkgs ? import <nixpkgs> {}, 
+  withMpv ? true,
+  withVlc ? false,
+  withIina ? false,
```

Comment:
are you sure these are defined given just this file? Seems a bit odd

Reply:
Or is that part of the upstream nixos-unstable package?

Reply:
They are arguments that u can pass. Its supported in the nix-unstable package of ani-cli 4.6.
see here [ani-cli nixpkg](https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/applications/video/ani-cli/default.nix#L57)